airflow/cli/commands/kerberos.py

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Kerberos command"""
+import daemon
+import rich_click as click
+from daemon.pidfile import TimeoutPIDLockFile
+
+from airflow import settings
+from airflow.cli import airflow_cmd, click_daemon, click_log_file, click_pid, click_stderr, click_stdout
+from airflow.configuration import conf
+from airflow.security import kerberos as krb
+from airflow.utils.cli import setup_locations
+
+
+@airflow_cmd.command()
+@click.argument("principal")
+@click_stdout
+@click_stderr
+@click_pid
+@click_daemon
+@click_log_file
+@click.option("-k", "--keytab", metavar="KEYTAB", help="keytab", default=conf.get("kerberos", "keytab"))
+def kerberos(principal, stdout, stderr, pid, daemon_, log_file, keytab):
+    """Start a kerberos ticket renewer"""
+    print(settings.HEADER)
```

Comment:
```suggestion
    from rich.console import Console
    console = Console()
    console.print(settings.HEADER)
```

Ideally all output would flow through `rich.console.Console`.

Reply:
Excellent, I'll remember that. Here's the before (bottom) and after (top)

<img width="1298" alt="Screen Shot 2022-06-17 at 8 52 26" src="https://user-images.githubusercontent.com/11639738/174196414-edf41825-d35c-42dc-ae9f-2ffee87152da.png">